es_ES/ui-performance/index.md

```diff
@@ -0,0 +1,404 @@
+
+<aside class="alert alert-info" markdown="1">
+**Por qué debe ejecutarse en un dispositivo real:**
+
+* Los simuladores y emuladores no utilizan el mismo hardware, por lo que sus características de rendimiento son diferentes&mdash; algunas operaciones son más rápidas en los simuladores que en los dispositivos reales, y otras son más lentas.
+* El modo depuración permite realizar comprobaciones adicionales (como asserts) que no se ejecutan en compilaciones profile o release, y estas comprobaciones pueden ser costosas.
+* El modo de depuración también ejecuta el código de una manera diferente que el modo release. La construcción del debug compila el código de Dart "just in time" (JIT) a medida que la aplicación se ejecuta, pero las construcciones de profile y release se precompilan a instrucciones nativas (también llamadas "ahead of time", o AOT) antes de que la aplicación se cargue en el dispositivo. JIT puede hacer que la aplicación se pause para la compilación de JIT, lo que a su vez puede causar jank.
+
+</aside>
+
+### Ejecutar en modo profile
+
+El modo profile de Flutter compila e inicia tu aplicación de forma casi idéntica al modo release, pero con la funcionalidad adicional suficiente para permitir la depuración de problemas de rendimiento.
+Por ejemplo, el modo de perfil proporciona información de trazabilidad al
+[Observatory](https://dart-lang.github.io/observatory/) y otras herramientas.
+
+Lanza la aplicación en modo profile de la siguiente manera:
+
+{% comment %}
+Aún no disponible en VS Code.
+{% endcomment %}
+
+<ul markdown="1">
+<li markdown="1">En Android Studio e IntelliJ, usa el elemento del menú
+    **Run > Flutter Run main.dart in Profile Mode**.
+</li>
+<li markdown="1">En VS Code, abre tu archivo `launch.json`, y asigna la propiedad
+`flutterMode` a `profile`
+(cuando termines el profile, cámbialo de nuevo hacia `release` o `debug`):
+
+```
+"configurations": [
+	{
+		"name": "Flutter",
+		"request": "launch",
+		"type": "dart",
+		"flutterMode": "profile"
+	}
+]
+```
+</li>
+<li markdown="1">Desde la línea de comando, usa el parámetro `--profile`:
+
+{% prettify sh %}
+$ flutter run --profile
+{% endprettify %}
+</li>
+</ul>
+
+Para obtener más información sobre cómo funcionan los diferentes modos, consulta
+ [Modos en Flutter](https://github.com/flutter/flutter/wiki/Flutter%27s-modes).
+
+Comenzarás activando la ventana de rendimiento, como se explica en la siguiente sección.
+
+## La ventana de rendimiento
+
+La ventana de rendimiento muestra las estadísticas en dos gráficos que muestran dónde se está gastando el tiempo tu aplicación.
```

Comment:
The performance overlay displays statistics ...
La capa sobrepuesta de rendimiento muestra estadísticas ...

Reply:
Yo la había quitado la palabra "sobrepuesta" dado que es el funcionamiento normal de cualquier ventana. De otra parte, es una ventana (se podría decir que es una ventana de rendimiento sobre una capa, pero sería muy largo y aparece varias veces en el documento) que despliega información a cerca del rendimiento de manera gráfica.